org.eclipse.lemminx/src/test/java/org/eclipse/lemminx/AbstractCacheBasedTest.java

```diff
@@ -37,7 +39,13 @@ public abstract class AbstractCacheBasedTest {
 
 	static {
 		try {
-			parentDir = Files.createTempDirectory("lemminx");
+			String cacheInRepoDir = System.getProperty("cacheInRepoDir");
```

Comment:
Slight preference for something like `lemminx.cacheInRepoDir`, just so that it's properly namespaced and unlikely to clash with anything (though it would be very rare).

Reply:
Also, just realized that